src/features/collaborator-avatars/CollaboratorAvatarItem.js

```diff
@@ -44,14 +46,14 @@ const CollaboratorAvatarItem = (props: Props) => {
         ) : (
             <Avatar name={name || '-'} {...rest} isExternal={isExternalCollab} shouldShowExternal={allowBadging} />
         );
-
+    const expirationMessage = intl.formatMessage(messages.expirationText);
     const expirationBadge =
         allowBadging && expiration && expiration.executeAt ? (
             <Tooltip
                 position="middle-right"
                 text={<FormattedMessage {...messages.expirationTooltipText} values={{ date: expiration.executeAt }} />}
```

Comment:
can you update the tooltip `text` prop to use an `intl.formatMessage` instead of `<FormattedMessage />`? The latter creates a <span>, and is likely the reason why the `describedBy` doesn't disappear